src/types/UiaTextRangeBase.cpp

```diff
@@ -1289,9 +1289,9 @@ try
         }
     }
 
-    FAIL_FAST_IF(!(newViewport.Top >= topRow));
-    FAIL_FAST_IF(!(newViewport.Bottom <= bottomRow));
-    FAIL_FAST_IF(!(_getViewportHeight(oldViewport) == _getViewportHeight(newViewport)));
+    assert(newViewport.Top >= topRow);

```

Comment:
@lhecker, can you suggest something better than just assert? If there is anything better. Otherwise, let's just do this.

Reply:
The thread I dug up from like 3 years ago had Michael suggesting telemetry asserts. Unfortunately, that requires a decent amount of work to set up.

Reply:
The only alternative to telemetry asserts that come to my mind is `WI_ASSERT`.